library/pkcs11_client.c

```diff
@@ -265,12 +265,88 @@ static int pkcs11_sign( void *ctx_arg,
     return( pkcs11_err_to_mbedtls_pk_err( rv ) );
 }
 
+static int pkcs11_verify( void *ctx_arg,
```

Comment:
This function has some code in common with `pkcs11_sign`. Can it be shared in an auxiliary function?